README.md

```diff
@@ -0,0 +1,180 @@
+  
+## Importing machine(s) into GLPI
+The `population/create...`  scripts handle creating a computer and updating fields in a GLPI deployment. For more specific usage information use the help message provided by the scripts.
+
+### RHEL, CentOS, and Fedora workflow (Directly on Target CLI):
+![alt text](/docs/images/GLPI-Direct-REST.png)
+
+NOTE: This approach must have direct CLI access to the target server, the ability to host this code on the target, and may need sudo on the target
+1. Copy the repository to the server to be imported.
+2. Ensure you have the following Python requirements installed:
+    - Python3
+    - pip3
+    - pexpect
+3. Ensure you have the following operating system requirements installed:
+    - lshw
+    - hostnamectl
+    - dmidecode
+    - lscpu
+    - uname
+    - ifconfig
+    - parted
+4. Run the script either with sudo or as a root user. Pass in the GLPI API token, general configuration, as well as the IP address of the GLPI instance.
+    - The API token is available from the Administration/Users, in the "Remote access keys" section of a specific user.
+    - An example of the general configuration YAML specifying a mapping of processing accelerator device ID to the name field of Processing Accelerators is as follows:
+```
+ACCELERATOR_IDS:
+  '0d5c': 
+    'ACC100' 
+  '0d8f': 
+    'N3000'
+```
+    - Optionally, pass in a switch configuration in the following format:
+```
+LAB:                     # lab name
+  switches:
+    IP:                  # IP address
+      name: switch_name  # switch name
+      location: location # location
+      username: username # username
+      password: password # password
+      type: type         # type of switch (currently supports dell or cumulus)
+
+      # Will result in {'lab': {'switches': {'ip': {'name: 'switch_name', 'location': 'location', 'username': 'username', 'password': 'password', type: 'type'}, {'ip_2':...}}}
+
+```
+    - For additional usage options see the help option of the script.
+6. If the script was run successfully, go to the GLPI URL and ensure that fields were correctly populated.
+7. If desired, manually set the location, technician, group, status, and any other desired fields for this machine, as these fields may help with identification or reporting later.
+8. If there was previously information populated, check fields are remove redundant information if present(for example, in the 'Components' section one may need "Unspecified" memory if previously populated). This has been specifically noticed as a remnant of running the Redfish based script, and one work around is currently to simply remove the machine if populating it a second time.
+
+### OS Agnostic, IPMI and Redfish based workflow (Remote):
+![alt text](/docs/images/GLPI-Remote-REST.png)
```

Comment:
Nitpicky, but the script wouldn't necessarily be run on a jumphost. Maybe replace with a more general term, like "Local Machine"?

Reply:
@adhil0 Please check the most recent commit, figure should be updated